avro-builder/tests/tests-allavro/src/test/java/com/linkedin/avroutil1/builder/SpecificRecordTest.java

```diff
@@ -118,8 +118,8 @@ private Object[][] TestRoundTripSerializationProvider() {
 
         {charseqmethod.TestCollections.class, charseqmethod.TestCollections.getClassSchema()},
```

Comment:
since the following lines are commented out, shouldn't the `,` be removed from this line too?